docs/governance/politeia.md

```diff
@@ -19,7 +19,7 @@ There is a fee for submitting a proposal (0.1 DCR), to limit the potential for p
 
 Reddit-style **up/down voting** is used for **proposal and comment sorting only**. Up/down votes are not anonymous.
 
-Ticket-voting on proposals doesn’t happen directly on Politeia, but within [Decrediton](https://docs.decred.org/getting-started/user-guides/decrediton-setup.md) or [politeiavoter CLI](https://github.com/decred/politeia/tree/master/politeiavoter) with dcrwallet
+Ticket-voting on proposals doesn’t happen directly on Politeia, but within [Decrediton](/getting-started/user-guides/decrediton-setup/) or [politeiavoter CLI](https://github.com/decred/politeia/tree/master/politeiavoter) with dcrwallet
```

Comment:
Some variation of this might be good to include:

Ticket-voting on proposals doesn't happen directly on Politeia, but within Decrediton or using politeiavoter CLI with dcrwallet.  Regardless of which voting client you use, your wallet signs the Politeia votes with the private keys that were used to purchase your Decred tickets and then sends the signed votes to the Politeia server to be recorded.  Vote are not associated with your Politeia account and politeiavoter CLI has additional privacy features that can be used to help keep the voting process anonymous.